src/status_im/contexts/communities/channel_details/view.cljs

```diff
@@ -0,0 +1,183 @@
+(ns status-im.contexts.communities.channel-details.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im.common.contact-list-item.view :as contact-list-item]
+    [status-im.common.contact-list.view :as contact-list]
+    [status-im.common.home.actions.view :as actions]
+    [status-im.constants :as constants]
+    [status-im.contexts.chat.group-details.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn group-chat-member-toggle
+  [member? selected? public-key]
+  (if-not member?
+    (if selected?
+      (rf/dispatch [:select-participant public-key true])
+      (rf/dispatch [:deselect-participant public-key true]))
+    (if selected?
+      (rf/dispatch [:undo-deselect-member public-key true])
+      (rf/dispatch [:deselect-member public-key true]))))
+
+(defn add-member-contact-item-render
+  [{:keys [public-key] :as item} _ _ {:keys [group]}]
+  (let [current-pk         (rf/sub [:multiaccount/public-key])
+        {:keys [contacts]} group
+        member?            (contains? contacts public-key)
+        checked?           (reagent/atom member?)]
```

Comment:
I believe we agreed for all new code (particularly chat related piece) we would use react state instead 👍 
cc @flexsurfer